wgsl/index.bs

```diff
@@ -5540,6 +5552,10 @@ See [[#sync-builtin-functions]].
            [0,|N|-1], then the expression evaluates to an [=invalid memory
            reference=].
 
+           If |i| is a signed integer, and |i| is less than 0:<br>
```

Comment:
Note this is slightly different from the other cases since the upper bound is not fixed until runtime.